src/pages/lesson-details/LessonDetails.tsx

```diff
@@ -78,6 +89,50 @@ const LessonDetails = () => {
     })
   }
 
+  const getCooperation = useCallback(() => {
+    return cooperationService.getCooperationById(id)
+  }, [id])
+
+  const { data: cooperation } = useQuery({
+    queryFn: getCooperation,
+    queryKey: ['cooperation', id]
+  })
+
+  useEffect(() => {
+    if (!cooperation?.sections) return
+
+    cooperation?.sections?.some((section) => {
```

Comment:
You already checked that in 102 line
```suggestion
    cooperation.sections?.some((section) => {
```

Reply:
done